tests/test_osfs.py

```diff
@@ -88,6 +89,11 @@ def test_expand_vars(self):
         self.assertNotIn("TYRIONLANISTER", fs2.getsyspath("/"))
 
     @pytest.mark.skipif(osfs.sendfile is None, reason="sendfile not supported")
+    @pytest.mark.skipif(
+        sys.version_info >= (3, 8),
```

Comment:
Does it matter that fs/osfs.py is comparing `sys.version_info[:2]` but `tests/test_osfs.py` is comparing `sys.version_info` ?

Reply:
@lurch : I don't think so, tuple comparison will stop on the smallest tuple.
```
>>> (3,8,0) >= (3,8)
True
>>> (3,8,1) >= (3,8)
True
>>> (3,8,3) >= (3,8)
True
>>> (3,8) >= (3,8)
True
```

Reply:
Yes, that is why i didn't slice `version_info`.

Reply:
Cool, thanks for confirming :+1: